### ISSUE TITLE
Loss decline is extremely unstable

### ISSUE THREAD

**Issue by PAINSGAINSL** [Megvii-BaseDetection/YOLOX#575]
Hi, I have a question. When I was training my data set, I found that loss concussion was severe. Is this a normal phenomenon? I use the yolox_m , turned off data enhancement，batchsize = 4. The specific Settings are as follows：
        
        self.num_classes = 1
        self.depth = 0.67
        self.width = 0.75
        self.exp_name = os.path.split(os.path.realpath(__file__))[1].split(".")[0]

        # ---------------- dataloader config ---------------- #
        # set worker to 4 for shorter dataloader init time
        self.data_num_workers = 0
        self.input_size = (512, 512)
        self.random_size = (16, 16)
        self.train_ann = "instances_train2017.json"
        self.val_ann = "instances_val2017.json"

        # --------------- transform config ----------------- #
        self.mosaic_prob = 1.0
        self.mixup_prob = 1.0
        self.degrees = 10.0
        self.translate = 0.1
        self.scale = (0.1, 2)
        self.mscale = (0.8, 1.6)
        self.shear = 2.0
        self.perspective = 0.0
        self.enable_mixup = False

        # --------------  training config --------------------- #
        self.warmup_epochs = 5
        self.max_epoch = 300
        self.warmup_lr = 0
        self.basic_lr_per_img = 0.01 / (64.0)
        self.scheduler = "yoloxwarmcos"
        self.no_aug_epochs = 300
        self.min_lr_ratio = 0.05
        self.ema = True
        self.weight_decay = 5e-4
        self.momentum = 0.9

        # -----------------  testing config ------------------ #
        self.test_size = (512, 512)
        self.test_conf = 0.01
        self.nmsthre = 0.65
![Snipaste_2021-08-23_19-09-46](https://user-images.githubusercontent.com/62642906/130714509-0d51f8e7-7766-44d7-abf2-f9973065de7a.png)

**Comment by Joker316701882** [comment]
@PAINSGAINSL It is based on the quality of your datasets. For a high-quality dataset, I think the loss should be relatively stable. By the way, what do you think about the final mAP (71.7%)?

**Comment by PAINSGAINSL** [comment]
This result is not good , and other networks can be used to get better results. Such as, for the same data set, mAP>90 on LFFD. Otherwise, in other networks, Loss does not show such a phenomenon of oscillation.  So, I wonder if there's something wrong.

> @PAINSGAINSL It is based on the quality of your datasets. For a high-quality dataset, I think the loss should be relatively stable. By the way, what do you think about the final mAP (71.7%)?

**Comment by ruinmessi** [comment]
You can try to add mixup aug and reduce your lr and final lr with self.basic_lr_per_img  and self.min_lr_ratio.  And I think batchsize=4 is somewhat too small. try to increase it if you have enough GPU mem.
